pythonforandroid/recipes/pyzmq/__init__.py

```diff
@@ -13,7 +13,7 @@ class PyZMQRecipe(CythonRecipe):
     version = 'master'
     url = 'https://github.com/zeromq/pyzmq/archive/{version}.zip'
```

Comment:
Could we take the opportunity to pin this version to last release so our builds are more reproducible?
```suggestion
    version = '20.0.0'
    https://github.com/zeromq/pyzmq/archive/v{version}.zip
```

Reply:
Changed